tools/ci_build/github/azure-pipelines/azure-pipelines-py-packaging.yml

```diff
@@ -178,7 +178,9 @@ jobs:
       workingFolder: '$(Build.BinariesDirectory)'
 
   - script: |
-     python -m pip install -q pyopenssl setuptools wheel numpy==1.16.6
+     python -m pip install --upgrade pip
+     python -m pip uninstall -y -q onnxruntime 2>pip.log
```

Comment:
will pip.log ever be read? looks like everything will be cleaned up later by clean-agent-build-directory-step.yml. if not, how about just redirecting to /dev/null?

also, have you considered using script's failOnStderr setting?

Reply:
But failOnStderr is default false, so it confused me.

This is Windows, I can't use /dev/null.

Reply:
Windows supports `command > null 2>&1` if you want to go that route.